src/azure-cli/azure/cli/command_modules/network/commands.py

```diff
@@ -777,9 +782,25 @@ def _make_singular(value):
         g.generic_update_command('update', child_collection_prop_name='inbound_nat_pools',
                                  custom_func_name='set_lb_inbound_nat_pool')
 
-    with self.command_group('network lb address-pool', network_lb_sdk) as g:
+    with self.command_group('network lb address-pool', network_lb_backend_pool_sdk) as g:
+        g.custom_command('create', 'create_lb_backend_address_pool')
+        g.show_command('show', 'get')
+        g.command('list', 'list')
+        g.custom_command('delete', 'delete_lb_backend_address_pool')
```

Comment:
better to add `confirmation=True`?

Reply:
it already existed before this PR. I don't want to include breaking change.